CDCgov/prime-reportstream
# Experience-8108: Remove upload-to-send functionality

We got the go-ahead from O+O to remove the upload-to-send functionality since that's been moved over to SimpleReport.

This strictly removing the _front-end_ logic for upload-to-send so as to make the public validator work slightly easier for us.  Nothing on the API is changing as a part of this changeset.

Test Steps:

CSV Sender
1. Impersonate an Organization with a Sender whose format is CSV
2. Go to http://localhost:3000/file-handler/validate
3. Ensure the copy is all correct around file format
4. Try uploading file and ensure success/failure is correct
<img width="1299" alt="Screenshot 2023-01-30 at 14 22 11" src="https://user-images.githubusercontent.com/2421042/215575668-ab89f22a-08c1-4117-8ee4-c0c6355cf4c0.png">
<img width="1299" alt="Screenshot 2023-01-30 at 14 22 18" src="https://user-images.githubusercontent.com/2421042/215575672-3a97d9f1-5673-420f-bba9-89d306ba4b6b.png">
<img width="1299" alt="Screenshot 2023-01-30 at 14 22 20" src="https://user-images.githubusercontent.com/2421042/215575676-1c45cbf2-c519-4744-9a5c-66c7cffd0097.png">

HL7 Sender
1. Impersonate an Organization with a Sender whose format is HL7
2. Go to http://localhost:3000/file-handler/validate
3. Ensure the copy is all correct around file format
4. Try uploading file and ensure success/failure is correct
<img width="1299" alt="Screenshot 2023-01-30 at 14 22 50" src="https://user-images.githubusercontent.com/2421042/215575832-9ef3727a-4c1d-4721-806d-26e76da1d4ad.png">
<img width="1159" alt="Screenshot 2023-01-30 at 14 26 15" src="https://user-images.githubusercontent.com/2421042/215575838-82ef07c8-94f3-4598-b425-7fc41e51c67c.png">
<img width="1299" alt="Screenshot 2023-01-30 at 14 22 58" src="https://user-images.githubusercontent.com/2421042/215575837-0d838a23-2c9b-48eb-8e47-782efda4ce85.png">

### Testing
<strike>- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?</strike>

## Linked Issues
- Fixes #8108
